Specs/spec-main.js

```diff
@@ -136,6 +136,8 @@
 
         window.it = function(description, f, timeout, categories) {
             originalIt(description, function(done) {
+                // Clear the RequestScheduler before for each test in case requests are still active from previous tests
+                Cesium.RequestScheduler.clearForSpecs();
```

Comment:
I'll see what I can do, because I don't like this either. Maybe this just needs to be called for specs that are doing more fine-grained request checking.